tests/test.mjs

```diff
@@ -210,13 +210,18 @@ let old_coapp;
 }
 
 if (!old_coapp) {
-  const codecs = await exec("codecs");
-  assert_deep_equal("codecs", codecs, expected_codecs);
-  const formats = await exec("formats");
-  if (os.platform() == "darwin") {
-    assert_deep_equal("formats", formats, expected_formats);
+  const use_prebuilt_ffmpeg = await exec("use_prebuilt_ffmpeg");
+  if (use_prebuilt_ffmpeg) {
```

Comment:
Add an option to the test (the same way we have --with-network). `--no-ffmpeg-codecs-tests`. And pass this option to the test in the test.yaml file.

Reply:
Added.